### PR TITLE
Test against dalli 3.0

### PR DESCRIPTION
**Pull request by zarqman**
This adds testing support for dalli 3.0.

Precursor to #556.

### CODE REVIEW

**Review by grzuy — COMMENTED** [review]
Thank you very much for this!

From what I see in the diff there were no changes needed to make test pass against dalli 3. That means technically dalli 3 was supported, but not tested against, right?

If so, for the sake of clarity, can we please change the commit message and PR title to "Test against dalli 3"?

Thanks!

File: spec/acceptance/stores/active_support_dalli_store_spec.rb [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 require_relative "../../spec_helper"
 
-if defined?(::Dalli)
+should_run =
+  defined?(::Dalli) &&
+  Gem::Version.new(::Dalli::VERSION) < Gem::Version.new("3")
+
+if should_run
```
[the remark this answers is not in the document]
Comment (zarqman) [reply spec/acceptance/stores/active_support_dalli_store_spec.rb:9]: When other tests are conditional upon a gem version, they all use `should_run` in the same way. I copied the style directly from those with the idea of maintaining consistency across the specs. See active_support_redis_cache_store_pooled_spec.rb or active_support_redis_cache_store_spec.rb.

If you still want it changed to be more direct, happy to do so.

**Comment by zarqman** [comment]
PR title and commit message updated to reflect this is just about testing.

File: spec/acceptance/stores/active_support_dalli_store_spec.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (grzuy) [reply spec/acceptance/stores/active_support_dalli_store_spec.rb:9]: Ah, didn't remember that :-)

Consistency for the scope of this changes sounds good to me.

If you feel like opening separate PR to refactor/clean up all those `should_run` local variables, go for it.

Thanks!

**Pull request merged**